packages/sanity/src/desk/panes/document/documentPanel/documentViews/FormView.tsx

```diff
@@ -136,7 +148,7 @@ export function FormView(props: FormViewProps) {
       width={1}
     >
       <PresenceOverlay margins={margins}>
-        <Box as="form" onSubmit={preventDefault} ref={formRef}>
+        <Box as="form" onSubmit={preventDefault} ref={setRef} style={{position: 'relative'}}>
```

Comment:
Not sure if this is in the a hot path performance wise, but worth moving the style to the outer scope here?

Reply:
Ah! that was me debugging and never removed it. That style doesn't need to be there I will remove it. great catch thanks!